src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClient.java

```diff
@@ -22,29 +73,154 @@
    * @return ActionPlanDTO representation of the created action plan
    * @throws RestClientException for failed connection to action service
    */
-  ActionPlanDTO createActionPlan(String name, String description, HashMap<String, String> selectors)
-      throws RestClientException;
+  @Retryable(
```

Comment:
The `@Retryable` annotation should be used very sparingly. We can't have users waiting 25 seconds for a response.